File: src/main/java/custom/tibame201020/adbOpenCv/script/gearScript/GearImageDTOs.java
package custom.tibame201020.adbOpenCv.script.gearScript;

public interface GearImageDTOs {

    record GearRegion(
            int x,
            int y,
            int width,
            int height
    ) {
    }

    record GearOcr(
            String ocrTemplatesPath,
            GearRegion gearRegion,
            double threshold
    ) {
    }

}

File: py-version/script/gearScript/gear_image_dtos.py
"""
Gear 影像 DTOs
"""
from dataclasses import dataclass


@dataclass
class GearRegion:
    """裝備區域"""
    x: int
    y: int
    width: int
    height: int


@dataclass
class GearOcr:
    """裝備 OCR 配置"""
    ocr_templates_path: str
    gear_region: GearRegion
    threshold: float









